xtask/src/tidy.rs

```diff
@@ -147,6 +147,7 @@ MIT/Apache-2.0
 MPL-2.0
 Unlicense OR MIT
 Unlicense/MIT
+Zlib
```

Comment:
This is coming from `foldhash`, which was introduced in [hashbrown 0.15](https://github.com/rust-lang/hashbrown/blob/master/CHANGELOG.md#v0150---2024-10-01).